setup.cfg

```diff
@@ -28,6 +28,7 @@ project_urls =
 
 [options]
 packages = find:
+include_package_data = True
```

Comment:
This one is actually unnecessary because you had it a few lines below, under the `install_requires`. So you can remove it.